datalad/local/add_readme.py

```diff
@@ -82,6 +90,31 @@ def __call__(dataset, filename='README.md', existing='skip'):
         # unlock, file could be annexed
         if lexists(filename):
             dataset.unlock(filename)
+        if not lexists(filename):
+            if isinstance(dataset.repo, AnnexRepo):
+                # if we have an annex repo, shall the README go to Git or annex?
+                attr_state = dataset.status(
+                        '.gitattributes',
+                        result_renderer='disabled',
+                        return_type='item')
+                if attr_state:
+                    # we have made sure that .gitattributes exists. We only fail
+                    # now if the state of the .gitattributes file is not clean
+                    if not attr_state[0].get('state') == 'clean':
+                        raise RuntimeError(
+                            'Stopping, as the .gitattributes file is modified'
```

Comment:
```suggestion
                            'Stopping, as the .gitattributes file is %s', attr_state[0].get('state')
```

Reply:
^ Nitpicky, but while at it ... would become another UX issue eventually if we reported "modified" here, when it's "untracked".